kolibri/plugins/setup_wizard/assets/src/views/LodJoinFacility.vue

```diff
@@ -90,6 +91,18 @@
           }
         });
       },
+      goToSignIn() {
+        this.wizardService.send({
+          type: 'SIGN_IN_INSTEAD',
+          value: {
+            importDeviceId: this.wizardService.state.context.importDeviceId,
```

Comment:
[Non blocker] I was just thinking that If we already have these values in the state context, then I wonder whether its needed to assign them again? I think that the only value we would need is the `importOrJoin` prop, but wanted to know your thoughts about this.

Reply:
The type `SIGN_IN_INSTEAD` represents the idea of skipping back to `selectLodSetupType`, and moving through `selectLodFacility`, before the machine is being sent to `lodProceedJoinOrNew`. 

`selectLodSetupType` uses the action `setLodType` which needs the `importDeviceId` and the `importOrJoin` properties, and `selectLodFacility` uses the action `setSelectedImportDeviceFacility`, which sets the other included props.

So including these previously used actions is mainly to reduce the risk of those props being reset because the properties are not being reselected before the machine is being sent to `lodProceedJoinOrNew`.

Reply:
Oh okie, thank you!